pyqt-apps/siriushla/widgets/spinbox.py

```diff
@@ -163,6 +163,15 @@ def wheelEvent(self, event):
         else:
             super().wheelEvent(event)
 
+    def focusOutEvent(self, event):
```

Comment:
If I understand this diff correctly this can be  potentially major change, depending on how each person uses the GUI. It would be advisable to inform specially the operators about this change before deploy, if it did come as a request from them or if they are still not aware of it.